package.json

```diff
@@ -20,8 +20,8 @@
   },
   "license": "Apache-2.0",
   "author": "StarkWare Industries Ltd.",
-  "main": "dist/src/js/index.js",
-  "types": "dist/src/js/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/types/src/js/index.d.ts",
```

Comment:
maybe it's worth moving the types to `dist/types/index.d.ts` or `dist/index.d.ts`, without the `src/js`, but not critical.